.github/workflows/build-core-image.yml

```
@@ -0,0 +1,222 @@
+  #          -d '{"jsonrpc":"2.0","id":1,"method":"openhuman.about_app_list","params":{}}' \
+  #          http://<node-b-ip>:7788/rpc
+  #   4. Add the four secrets listed above to GitHub Actions.  Subsequent
+  #      pushes to main will deploy automatically.
+  # ===========================================================================
+  deploy-node-b:
+    name: Deploy to Node B via Komodo
+    runs-on: ubuntu-22.04
+    needs: build-and-push
+    if: >-
+      ${{
+        github.ref == 'refs/heads/main'
```

actionlint (1.7.12)

[error] context "secrets" is not allowed here. available contexts are "github", "inputs", "needs", "vars". see https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability for more details

(expression)